tools-image/build-arm-image.sh

```diff
@@ -335,7 +335,7 @@ if [ -z "$EFI" ]; then
   exit 1
 fi
 
-cp -rfv /arm/grub/efi/* $EFI
+cp -rfv /efi/* $EFI
```

Comment:
how come this changed?

Reply:
Because before we need to pick the arm packages and the x86 packages from the direct link and store them in a different place to differentiate them. Now with luet we install from the repo so it will pick the proper arch automatically, so we can store them in the same place and jsut use one dir.

So running the tools-image from x86 -> /efi will contain the x86 efi artifacts
runint the tools-image from arm64 -> /efi will contain the arm64 efi artifacts

Reply:
nice